doc-tool/src/dotty/tools/dottydoc/staticsite/Site.scala

```diff
@@ -167,9 +167,9 @@ case class Site(
   private def defaultParams(pageLocation: JFile, additionalDepth: Int = 0): DefaultParams = {
     val pathFromRoot = stripRoot(pageLocation)
     val baseUrl: String = {
-      val rootLen = root.getAbsolutePath.split(sep).length
-      val assetLen = pageLocation.getAbsolutePath.split(sep).length
-      "../" * (assetLen - rootLen - 1 + additionalDepth) + "."
+      val rootLen = root.toPath.normalize.getNameCount
+      val assetLen = pageLocation.toPath.normalize.getNameCount
```

Comment:
Don't you need to convert to absolute path as well?

Reply:
Usage of absolute path info in URL seems problematic. For now, I added `toAbstractPath` to keep the logic as before.